app/uk/gov/hmrc/childcarecalculatorfrontend/views/partnerMinimumEarnings.scala.html

```diff
@@ -36,7 +36,7 @@
 )(implicit request: Request[_], messages: Messages)
 
 @title = @{
-    if(form.errors.nonEmpty) messages("site.title.error", messages("partnerMinimumEarnings.title", amount)) else messages("partnerMinimumEarnings.title", amount)
+    if(form.errors.nonEmpty) messages("site.title.error", messages("partnerMinimumEarnings.form", amount)) else messages("partnerMinimumEarnings.form", amount)
```

Comment:
I am afraid, this is not the right way to do it. For the machine it does not matter, but for us, developers, it immediately raise questions, like: "what was the reason there is 'form' message key in place of title?".

AFAIU, the text you needed was already present under this message key. But instead of putting the key with "form" into title and heading below, I'd advise to leave the keys as they are and change the messages under these keys.